apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -48,7 +48,10 @@ const errorMessages = {
 
 const bountiesFor = ({ bountySettings, issues, tokens }) => issues.reduce(
   (bounties, issue) => {
+    const data = issue.repository.decoupled ? issue : undefined
```

Comment:
call here just to ensure that `undefined` is the appropriate type for non-decoupled repos data field